.gitmodules

```diff
@@ -1,5 +1,5 @@
 [submodule "dotCMS/src/main/enterprise"]
 	path = dotCMS/src/main/enterprise
 	url = git@github.com:dotCMS/enterprise-2.x.git
-        branch = master
+	branch = 18893-exclude-system-host-folder-in-bundle
```

Comment:
should be this merge?

Reply:
No, this will be changed once the EE branch get merged.